cpt-core/src/cptcore/functional/cca.py

```diff
@@ -37,8 +37,11 @@ def canonical_correlation_analysis(
         f_lon_dim=None, 
         f_sample_dim=None, 
         f_feature_dim=None, 
-        **kwargs
+        **_
```

Comment:
Here I'm just following the convention of using the name `_` for a variable that is intentionally never used. This silences an unused variable warning from my IDE.